app/templates/components/paper-item.hbs

```diff
@@ -1,11 +1,3 @@
-{{#if action}}
-  {{#paper-button class="md-no-style" noink=true onClick="buttonAction" skipProxy=true}}
-    <div class="md-list-item-inner">
-      {{yield}}
-    </div>
-  {{/paper-button}}
-{{else}}
-  <div class="md-no-style md-list-item-inner" {{action "buttonAction" on="click"}}>
-    {{yield}}
-  </div>
-{{/if}}
+<div class="md-no-style md-list-item-inner" {{action "buttonAction" on="click"}}>
```

Comment:
`<div class="md-no-style md-list-item-inner" {{action "buttonAction"}}>` also works fine. Should we stick to it?